src/OnlineRetailPortal.Contracts/Contracts/IProductStoreFactory.cs

```diff
@@ -6,5 +6,6 @@ namespace OnlineRetailPortal.Contracts
 {
     public interface IProductStoreFactory
     {
+        IProductStore GetStoreType(string storeValue);
```

Comment:
should just be GetProductStore(string parameterName);

Reply:
done